tests/test_parsers.py

```diff
@@ -117,21 +118,10 @@ def test_parse_obographs(self):
             write_table(msdf, file)
         self.assertEqual(
             len(msdf.df),
-            9881,
+            8099,
```

Comment:
@cthoyt I reinstated this test. Analysing the warnings, I noticed that the 800 less parsed entities (which is good, since they are not in the prefix map) are due to https://github.com/biopragmatics/bioregistry/issues/917. 

I think this is entirely independent of this PR, so I will approve this, this is just FYI.